src/Commands/BuildCommand.php

```diff
@@ -23,9 +25,18 @@ public function handle(): void
                 echo $output;
             });
 
+        $os = $this->argument('os');
+
+        if ($os == null) {
```

Comment:
Could this be simplified to...?

```suggestion
        if (! $os = $this->argument('os')) {
```